docs/source/docs/quick-start/quick-install.md

```diff
@@ -0,0 +1,38 @@
+# Quick Install
+
+## Install the latest image of photonvision for your coprocessor
+
+- For the supported coprocessors
+  - RPI 3,4,5
+  - Orange Pi 5
+  - Limelight
+
+For installing on non-supported devices {ref}`see. <docs/advanced-installation/sw_install/index:Software Installation>`
+
+[Download](https://github.com/PhotonVision/photonvision/releases/latest) the latest preconfigured image of photonvision for your coprocessor
+
+| Coprocessor          | Image filename                                                                                                                                                                     | Jar                                                                                                                                                  |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| OrangePi 5           | [photonvision-{version}-linuxarm64_orangepi5.img.xz](https://github.com/PhotonVision/photonvision/releases/latest/download/photonvision-v2024.3.1-linuxarm64_orangepi5.img.xz)     | [photonvision-{version}-linuxarm64.jar](https://github.com/PhotonVision/photonvision/releases/latest/download/photonvision-v2024.3.1-linuxarm64.jar) |
```

Comment:
So this works great, except that the image name changes every release. LOL.

Reply:
ugh. right...

Reply:
why we do dat!

Reply:
I can't think of a great way to automate this, short of a redirect on our VPS. Let's move this table into our template for releases like how ascope does it, and delete the links from the docs to reduce confusion?